roles/syslog-client/tasks/main.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Ensure that apt-transport-https is installed
+  apt:
+    name: apt-transport-https
+    state: present
+
+- name: Add Elesticsearch key
+  apt_key:
+    url: https://artifacts.elastic.co/GPG-KEY-elasticsearch
```

Comment:
I'd prefer to copy the key into the repository.  That's the best way to make sure the key isn't tampered with.

Reply:
Hmm... and have it blow up when the key is rolled over?

Reply:
Yes, because you *want* to do key rollovers manually.  Otherwise, an attacker that has access to elestico's servers, could just as easily replace the gpg key with his own, in addition to changing the packages.